config/src/main/java/com/alibaba/nacos/config/server/utils/AccumulateStatCount.java

```diff
@@ -19,20 +19,24 @@
 import java.util.concurrent.atomic.AtomicLong;
 
 /**
- * Accumulate Stat Count
+ * Accumulate Stat Count.
  *
  * @author Nacos
  */
 public class AccumulateStatCount {
-    
+
```

Comment:
indent？ please check code style, it seems that the original is right

Reply:
I checked that my idea tool did not import nacos-code-style-for-idea.xml,  I will resubmit it later.